.bump-version.js

```diff
@@ -0,0 +1,56 @@
+const replace = require("replace");
+const yaml = require('js-yaml');
+const fs = require('fs');
+const {execSync} = require('child_process')
+
+const versionRegex = /\d+\.\d+\.\d+(-\w+)?/i;
+const newVersion = process.env.SEM_VER;
+
+function checkVersion(semver){
+
+    if(semver == undefined || semver == '') {
+        console.log("semver cannot be blank");
+        process.exit(1);
+    }
+
+    if(semver.match(versionRegex) == undefined) {
+        console.log(`semver: ${semver} must be Semantic Version <Major>.<Minnor>.<Patch>[-<type>], examples: '0.20.0', '0.20.0-rc1'`)
+        process.exit(1);
+    }
+
+    return semver;
+}
+
+// check is a valid version
+checkVersion(newVersion);
+
+console.log(`Bumping version to: ${newVersion}`);
+
+// bumping version using replace for 'packages/relay/package.json', 'packages/server/package.json', 'docs/openrpc.json'
+replace({
+    regex: '\"version\": \"\\d+\\.\\d+\\.\\d+(-\\w+)?\"',
+    replacement: `"version": "${newVersion}"`,
+    paths: ['packages/relay/package.json', 'packages/server/package.json', 'docs/openrpc.json'],
+    recursive: false,
+    silent: false,
+});
+
+// bump docker-compose.js version
+const dockerComposePath = 'docker-compose.yml';
+let dockerComposeYml = yaml.load(fs.readFileSync(dockerComposePath));
+dockerComposeYml.services.relay.image=`ghcr.io/hashgraph/hedera-json-rpc-relay:${newVersion}`
+fs.writeFileSync(dockerComposePath, yaml.dump(dockerComposeYml));
+console.log(dockerComposePath);
+
+// bump helm chart versions
+const helmChartPath = 'helm-chart/Chart.yaml';
+let doc = yaml.load(fs.readFileSync(helmChartPath, 'utf-8'));
+doc.version = newVersion;
+doc.appVersion = `${newVersion}`;
+fs.writeFileSync(helmChartPath, yaml.dump(doc));
+console.log(helmChartPath);
+
+// bump package-lock.json using npm itself, by running install it will update packages versions based on previous files
```

Comment:
there should be a bump for the `README.md` itself. 
There's a a link to the Open RPC spec.
When we branch release it should see a bump to the path e.g. https://github.com/hashgraph/hedera-json-rpc-relay/pull/951/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R18

This should only happen once so future bumps with the same major and minor versions would see no change

Reply:
handling README bumps now, only when is not an snapshot, as `main` branch has as "main" the version in the `README.md` file, similar to the `docker-compose.yml` file.